pkg/splunk/enterprise/names.go

```diff
@@ -70,7 +70,11 @@ const (
 
 	manualAppUpdateCMStr = "splunk-%s-manual-app-update"
 
-	applySHCBundleCmdStr = "/opt/splunk/bin/splunk apply shcluster-bundle -target https://%s:8089 -auth admin:`cat /mnt/splunk-secrets/password` --answer-yes -push-default-apps true"
+	applySHCBundleCmdStr = "/opt/splunk/bin/splunk apply shcluster-bundle -target https://%s:8089 -auth admin:`cat /mnt/splunk-secrets/password` --answer-yes -push-default-apps true &> %s &"
+
+	shcBundlePushCompleteStr = "Bundle has been pushed successfully to all the cluster members.\n"
+
+	shcBundlePushStatusCheckFile = "/opt/splunk/etc/shcluster_bundle_status.txt"
```

Comment:
Can we use a location somewhere from the /init-apps/, that way to avoid broader visibility of this file?

Reply:
I did not use that since in the future we were going to get rid of that location or at least change the name of that mount point.

Reply:
We better keep the Operator context out of the ../etc and ../var partitions. Otherwise, if Ansible/Docker/Operator keep adding, it will be kind of invading :-).  /init-apps/ is in our control and if it changes, we have to take care of this . Probably, we can offset from a global variable.